source/base/parameter_handler.cc

```diff
@@ -280,6 +297,45 @@ namespace Patterns
   }
 
 
+  void Integer::string_to_any(const std::string &s, boost::any &v) const
+  {
+    AssertThrow(match(s),
+                ParameterHandler::ExcValueDoesNotMatchPattern(s,description()));
+    if (v.type() == typeid(int *))
+      *(boost::any_cast<int *>(v)) = Utilities::string_to_int(s);
+    else if (v.type() == typeid(unsigned int *))
```

Comment:
Oh, I see -- you actually do. Does one need to document the kinds of types each pattern understands?